packages/sui-pde/src/components/experiment.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types'
+import useExperiment from '../hooks/useExperiment'
+
+export default function Experiment({children, ...props}) {
```

Comment:
I think that it could be interesting to destructure props and pass each one separately to `useExperiment` in order to filter not desired props. What do you think about that?

Reply:
sure! I agree